js/src/personal/jionghao/components/Work.module.css

```diff
@@ -28,3 +28,18 @@
   justify-content: flex-start;
   margin-bottom: 3vh;
 }
+
+.project{
```

Comment:
nit : feel free to fix later - white space here and elsewhere `.project {`

Reply:
You should turn on css autoformatting in prettier. Look at the tools-update channel in discord.